wagtail/wagtailusers/tests.py

```diff
@@ -297,6 +301,31 @@ def test_user_cannot_delete_self(self):
         # Check user was not deleted
         self.assertTrue(get_user_model().objects.filter(pk=self.deleter_user.pk).exists())
 
+
+    def test_user_cannot_escalate_privileges(self):
```

Comment:
This test doesn't belong in `TestUserDeleteViewForNonSuperuser`, as it's testing the 'edit' view, not the 'delete' view (and fiddling the `setUp` method to give change_user permission to deleters_group is breaking the intent of these tests, namely "a user with _only_ delete permission should be able to do these things").